core/src/com/unciv/logic/trade/TradeLogic.kt

```diff
@@ -139,6 +149,23 @@ class TradeLogic(val ourCivilization: Civilization, val otherCivilization: Civil
                     val nameOfCivToDeclareWarOn = offer.name
                     from.getDiplomacyManager(nameOfCivToDeclareWarOn).declareWar()
                 }
+                TradeType.PeaceDeclaration -> {
+                    val nameOfCivToDeclarePeaceOn = offer.name
+                    from.getDiplomacyManager(nameOfCivToDeclarePeaceOn).makePeace()
```

Comment:
You need agreement from both sides to make peace! This should be only "make a trade offer to make peace". Otherwise I can force the other player! 

Imagine a 3 player game, 2 humans one AI. You, human A, see that human B is about to take over a city. So what do you do? You trade the AI and say "take this money and make peace", and then there's peace, you've screwed the other human's plans! 
Same thing for AI. We should not be treating them differently.